java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Order/OrdersRepository.java

```diff
@@ -0,0 +1,178 @@
+    // a receipt to remind ourselves which entree it is and which order it is from).
+    // For Example:
+    // String sqlEntree = " INSERT INTO entree_ordered(order_id ,entree_id) VALUES (?,?) RETURNING *";
+    // EntreeOrdered entreeOrdered = jdbcTemplate.queryForObject(sqlEntree,
+    //   new BeanPropertyRowMapper<>(EntreeOrdered.class),
+    //   order.getId(),ordersRequestBody.getEntreeIds());
+//------ figure out how to return the list of appetizers & entrees
+// Get the List of appetizers:
+    // we are getting back the a List of appetizers ids, we need to get THE appetizer
+    //we use the method from the AppetizerRepository that we are getting the
+    // appetizer by their ids. We get the appetizer and put them on a new list
+    // For Example:
+    // List<Appetizer> appetizerOrders = new ArrayList<>();
+    // Appetizer appetizers = appetizerRepository.gettingAppetizerById(appetizerId);
+    // appetizerOrders.add(appetizers);
+// Get the List of entrees:
+    // we are getting back the List of entrees ids, we need to get THE entree
+    //we use the method from the EntreeRepository that we are getting the
+    // entree by their id. We get the entree and put them on a new list
+    // For Example:
+    // List<Entree> entreeOrders = new ArrayList<>();
+    // Entree entree = entreeRepository.gettingEntreeById(entreeId);
+    // entreeOrders.add(entree);
+// create a Entity:
+    // we need to return a list of appetizers & entrees(The name,cost etc)
+    //then we need to create a entity DTO(Data Transfer Object) to get the necessary fields
+    // we need to return to the customer(the id of the order, customerId and the list of
+    // appetizers & entrees(NOT THERE IDs) that the customer ordered).
+    // For Example:
+    //OrderDto orderDto = new OrderDto(order.getId(), ordersRequestBody.getCustomerId(),
+    // appetizerOrders, entreeOrders);
+//
+    public OrderDto CreateNewOrder(NewOrderRequestBody ordersRequestBody) {
+        // first when the order is created (Inserted into order) we need to find the appetizers
+        String sql = "INSERT INTO \"order\" (customer_id) VALUES (?) RETURNING *";
+        Orders order = jdbcTemplate.queryForObject(sql, new BeanPropertyRowMapper<>(Orders.class),
+                ordersRequestBody.getCustomerId());
+        log.info("Order is Created");
+        // the order is inserted now we take the appetizerId from the requestBody
+        // now we need to find the id by using the method in AppetizerRepository
+        log.info("Looking for the appetizers...");
+        List<Appetizer> appetizerOrders = new ArrayList<>();
+        List<Integer> listOfAppetizerIds = ordersRequestBody.getAppetizerIds();
+        System.out.println("List of appetizersIds: " + listOfAppetizerIds);
+        listOfAppetizerIds.forEach(appetizerId -> {
+            Appetizer appetizers =
+                    appetizerRepository.gettingAppetizerById(appetizerId);
```

Comment:
minor: you need to be more aware and careful with your naming schemes. You have a lot of singular/plural mistakes, like this `appetizers` here. This could lead to a lof of misunderstandings when reading your code

Reply:
I will keep that in mind